modules/pam_sepermit/pam_sepermit.c

```diff
@@ -372,13 +374,25 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags UNUSED,
 	char *level = NULL;
 	const char *cfgfile = SEPERMIT_CONF_FILE;
 
+#ifdef VENDORDIR
+	/* If the specified file does not exist, try the vendor file as fallback.
+	   This can be overruled again by a user provided file definition (conf=...) */
+	struct stat buffer;
+	if (stat(cfgfile, &buffer) != 0 && errno == ENOENT) {
+	  cfgfile = VENDORDIR"/security/sepermit.conf";
+	  pam_syslog (pamh, LOG_INFO,
+		      "cannot read settings from %s. Trying %s", SEPERMIT_CONF_FILE, cfgfile);
+	}
+#endif
+
```

Comment:
This shuldn't be done if the `conf=` option is specified.

Also, there is
```
pam_syslog(pamh, LOG_NOTICE, "Parsing config file: %s", cfgfile);
```
below, why would you want to issue this diagnostics at all?

Finally,
```
cfgfile = VENDORDIR"/security/sepermit.conf"
```
looks ugly after
```
const char *cfgfile = SEPERMIT_CONF_FILE;
```